sbin/jtreg.sh

```diff
@@ -92,7 +92,7 @@ settingUpEnvironmentVariablesForJTREG()
 runJtregViaMakeCommand()
 {
   echo "Running jtreg via make command (debug logs enabled)"
-  if [ -z $JTREG_TEST_SUBSETS }; then
+  if [ -z "$JTREG_TEST_SUBSETS" ]; then
     make test jobs=10 LOG=debug
```

Comment:
Not sure, my shell scripting skills are old :-).  But I used to use `"$..."` in order to resolve whitespace issues, but a few folks recently mentioned to me that `${...}` is the modern way to do that?